onnx_cntk/backend.py

```diff
@@ -0,0 +1,65 @@
+#-------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#--------------------------------------------------------------------------
+
+import numpy as np
+import cntk as C
+from onnx.backend.base import Backend
+from onnx import helper, TensorProto
+from .backend_rep import CNTKBackendRep
+
+
+class CNTKBackend(Backend):
+    @staticmethod
+    def set_device(device):
+        if device == 'CPU':
```

Comment:
Good point. Since cntk choses GPU as default if that's available, it may not be necessary to have a separate case. But I agree that it's a good idea to be explicit here, so I'll add another case for GPU/CUDA.

Reply:
Awesome!